cmd/snap/cmd_first_boot.go

```diff
@@ -20,12 +20,9 @@
 package main
 
 import (
-	"fmt"
-
 	"github.com/jessevdk/go-flags"
 
-	"github.com/ubuntu-core/snappy/i18n"
-	"github.com/ubuntu-core/snappy/snappy"
+	"github.com/ubuntu-core/snappy/overlord"
 )
 
 type cmdInternalFirstBoot struct{}
```

Comment:
Can we change that command to be `snap internal --firstboot` instead of `snap firstboot`? (not necessarily in this branch)